src/components/Card/Card.tsx

```diff
@@ -0,0 +1,112 @@
+import { useState, MouseEvent } from "react";
+import styles from "./card.module.css";
+import { formatDate, getTimeDifference } from "utils/helpers/dateUtils";
+import DeleteModal from "components/Modal/DeleteModal/DeleteModal";
+import FolderModal from "components/Modal/FolderModal/FolderModal";
+import noImage from "assets/images/bg_noImage.png";
+import meatballsIcon from "assets/images/ic_meatballs.svg";
+import starIcon from "assets/images/ic_star.svg";
+import { DELETE_LINK, ADD_LINK } from "utils/constants/strings";
+import { GetLinkResponse } from "types/apis";
+import { UseModal, Modal, OpenModal, CloseModal } from "utils/hooks/useModal";
```

Comment:
컴포넌트의 입장에서 import는 의존성이라고 볼 수 있는데요~
이 컴포넌트를 구성하기 위해 외부에서 가져와야만 하는 모듈들의 종류를 의미할 것 같고,
외부 의존성만 보고도 해당 컴포넌트가 어떤식으로 구성될 것인지 파악할 수가 있을 것 같아요~
그러면 import 구문도 종류별로 개행을 해주면 더욱 컴포넌트를 이해하는데 편리해질 것 같아요!

구분짓는 방식에 대해서는 일반적으로 어떻게 하는지도 찾아보고 수진님만의 규칙을 만들어봐도 좋을 것 같네요!
이러한 규칙을 eslint에서 강제할 수도 있습니다! [참고자료](https://pozafly.github.io/environment/putting-rules-into-import-syntax-with-eslint/)

```suggestion
import { useState, MouseEvent } from "react";

import DeleteModal from "components/Modal/DeleteModal/DeleteModal";
import FolderModal from "components/Modal/FolderModal/FolderModal";

import { GetLinkResponse } from "types/apis";

import { formatDate, getTimeDifference } from "utils/helpers/dateUtils";
import { DELETE_LINK, ADD_LINK } from "utils/constants/strings";
import { UseModal, Modal, OpenModal, CloseModal } from "utils/hooks/useModal";

import noImage from "assets/images/bg_noImage.png";
import meatballsIcon from "assets/images/ic_meatballs.svg";
import starIcon from "assets/images/ic_star.svg";

import styles from "./card.module.css";
```